src/lib.rs

```diff
@@ -12,6 +12,7 @@ pub mod config;
 pub mod market;
 pub mod models;
 pub mod possess;
+pub mod errors;
```

Comment:
I tend to prefer to sort things by what they accomplish, and not what they are, this can make it much easier to find things when the tree gets large enough.

Reply:
I just went to the bottom of the list and added it :)

Where would you have placed it?